biggraphite/drivers/elasticsearch.py

```diff
@@ -646,6 +705,7 @@ def _document_to_metric(self, document):
             read_on=ttls.str_to_datetime(document.read_on),
         )
 
+    @GET_METRIC_LATENCY.time()
```

Comment:
Why here and not on `get_metric`?

Reply:
__get_document is called by get_metric, __touch_document and __update_read_on. I wanted to catch all calls that read a metric.